index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/master.css" type="text/css">
+    <link rel="stylesheet" href="css/firstbox.css" type="text/css">
+    <link rel="stylesheet" href="css/secondbox.css" type="text/css">
+    <link rel="stylesheet" href="css/thirdbox.css" type="text/css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>RESUME</title>
+</head>
+
+
+<body>
+
+    <div class="header">
+        <div class="whitebox">
+            <div class="inner">
+                <p id="name">YOUR NAME</p>
+                <p id="role"> Software Developer</p>
+            </div>
+
+        </div>
+    </div>
+
+
+    <div class="content">
+
+        <div class="firstbox">
+            <div class="firstsubbox">

```

Comment:
Naming should be meaningful, also read about naming convention in html css and apply. Comment applicable throughout

Reply:
Done